### ISSUE TITLE
Kompatibilität mit "heroes-chronicles-fully-compability-hdmod"

### ISSUE THREAD

**Issue by Nihlo** [heroesiiifan/HoMM3-German#3]
Leider ist die deutsche Übersetzungs-Mod nicht mit dieser Chronicle-Interface-Mod kompatibel. 

Abgesehen davon, dass es die Mod bis jetzt nur in polnisch und englisch gibt, lässt sie sich auch nicht mit dem Installer des Autors mit den deutschen Chronicles installieren. Sehr schade, denn diese Interface(s) sind wirklich eine tolle Idee und Umsetzung. 

Mod findet sich hier:
https://www.moddb.com/mods/heroes-chronicles-fully-compability-hdmod

**Comment by heroesiiifan** [comment]
Prinzipiell definitiv möglich. Allerdings wohl nicht ganz so einfach. Weil es ja irgendwie alles so ein bisschen kombiniert (HD+, ERA, HC-Mod und die Chronicles selber).

**Comment by heroesiiifan** [comment]
Habe mal eine Uebersetzung dafuer gemacht. Ist nicht alles in Deutsch. War mir dann doch zu viel Aufwand. Im Readme steht, was fehlt.

Aber, ich denke, damit kann man ganz gut leben. Wer will, darf hier gerne unterstuetzen.

**Comment by heroesiiifan** [comment]
**Offene Punkte**

- [ ] Launcher (in .NET geschrieben; mit "dnlib" sollten sich Strings/Ressourcen anpassen lassen)
- [ ] Aufwendig gestaltete Buttons (z.B. "Play", aber auch "New Game") -> gibts teilweise in den deutschen Ressourcen
- [ ] Texte bei der Kampagnenauswahl
- [ ] Kampagnen Einführungen (Bilder) -> nicht in den Ressourcen vorhanden?!
- [ ] "The Glory of War"
- [ ] Previews extrahieren & Übersetzen
- [x] Tutorials aus Original extrahieren

**Comment by Nihlo** [comment]
Wow, das ging schnell. Vielen Dank
Leider lies sich bei mir der Installer gar nicht erst durchführen mit H3-Complete und schon vorhandener deutscher Chronicles von dir. Schätze ich muss nochmal alles deinstallieren, die Chronicles extra von GoG installieren, seinen Installere verwenden und danach erst deine Karten und diese neue Übersetzungen der Mod reinkopieren.

**Comment by heroesiiifan** [comment]
Ja, am besten den ganz normalen Installations-Weg (mit den englischen GOG-Chronicles) nehmen und wenn alles funktioniert noch die Dateien überschrieben. :)

**Comment by Darktobas** [comment]
hänge auch gerade an den Heroes Chronicles Kampagnen, da ich die immer mit in meine Hauptspiel integrieren wollte. Habe versucht, dies mit Project Chronicles umzusetzen, hat aber mäßig geklappt, Glory of War konnte ich nicht spielen und außerdem war da zu viel anderer kram drin (polnische Übersetzungen), was ich erstmal rausnehmen wollte, habe es aber dadurch kaputt gekriegt.
Im nächsten Ansatz habe ich die GoG-Versionen installiert, in einem Ordner zusammengefasst und dann in mein Heroes 3 Hauptverzeichnis eingepflegt. Mit meiner Retail-Version konnte ich die meisten Teile Übersetzen. Unterm Strich kann ich 6 Heroes Chronicles Teile komplett auf deutsch spielen (inklusive Sprachausgabe und Menü-Buttons. World Tree und Fiery Moon ist nie auf Deutsch erschienen, habe ich aber soweit ich konnte übersetzt. Der einzige Nachteil ist, dass ich nicht den HD-Mod habe, aber habe mich auch damit noch nicht beschäftigt.
Zu Glory of War habe ich ein ausführliches Lets Play gefunden. Damit kann man (wenn man viel Langeweile hat) die Kampagne nachbauen, inklusive Story und dabei auf alle Era-Features verzichten, um es für das klassische Heroes 3 kompatibel zu machen...

Falls du die deutschen Buttons und Texte brauchst bitte gerne melden.

**Comment by heroesiiifan** [comment]
Hallo @Darktobas:

Die deutschen Texte zu den Chronicles hab ich bereits. Ist in dem Paket auch drin (als Custom-Kampagne). Bzw. in der "heroes-chronicles-fully-compability-hdmod"-Übersetzung.

Die Buttons hätte ich auch. Ist aber nicht ganz so einfach zu integrieren, weil durch die andere Größe auch andere Koordinaten für die Buttons benötigt werden...

**Comment by Nihlo** [comment]
Heute Update der Mod auf 2.5.9 bzw. 1.0.1.9:
http://heroescommunity.com/viewthread.php3?TID=46774&PID=1572366#focus
https://www.moddb.com/mods/heroes-chronicles-fully-compability-hdmod/downloads/release-version-259

Hauptsächlich wohl die spanische Übersetzung und Fixes.

**Comment by Nihlo** [comment]
Update auf 2.6.0 / 1.0.2.0:

https://www.moddb.com/mods/heroes-chronicles-fully-compability-hdmod/downloads/release-version-260

Changelog bisher nur auf polnisch:
https://www.forum.acidcave.net/topic.php?TID=3892&pid=167213&page=29#167213
